azure-pipelines.yml

```diff
@@ -0,0 +1,27 @@
+trigger:
+  branches:
+    include:
+      - master
+      - refs/tags/*
+resources:
+  repositories:
+    - repository: internal-templates
+      type: github
+      name: xamarin/yaml-templates
+      endpoint: xamarin
+      ref: refs/heads/main
+variables:
+  - group: Xamarin-Secrets
+stages:
+  - stage: Build
+    jobs:
+    - job: buildWindows
+      pool:
+        vmImage: windows-latest
+      steps:
+      - task: MSBuild@1
+          displayName: 'Build Projects\OpenTK.Android\OpenTK.Android.csproj'
+          inputs:
+            solution: Projects\OpenTK.Android\OpenTK.Android.csproj
+            configuration: Release
```

Comment:
You could put `Configuration` under `variables`, like:

https://github.com/xamarin/Xamarin.Legacy.Sdk/blob/9428620338f5677cc54eeaa4d5ef714767352556/azure-pipelines.yml#L17-L18

Just so it would work for any future projects.